sdk/java-sdk/src/main/scala/kalix/javasdk/impl/action/ActionsImpl.scala

```diff
@@ -265,6 +265,7 @@ private[javasdk] final class ActionsImpl(
               // user stream failed with an "unexpected" error
               handleUnexpectedException(service, in, ex)
             }
+            .async
```

Comment:
Is this useful because it will allow each service/route to run upon a dedicated actor? Therefore each incoming request gets isolated by service type? i.e. Action, View, ESE, RESE, and VE?

Reply:
For the record if you're curious, this comes from [here](https://github.com/lightbend/kalix-jvm-sdk/issues/1078) and there was a nice and informative thread [here](https://typesafe.slack.com/archives/CQUASEUDV/p1661512698803409). But to quote Johan, I think the summary was:
> One client shared between all actors for all entity ids on the node -> one connection, all request sequenced.